spikeextractors/extractors/phyextractors/phyextractors.py

```diff
@@ -232,10 +242,13 @@ def write_sorting(sorting, save_path):
         spike_times = spike_times[sorting_idxs]
         spike_clusters = spike_clusters[sorting_idxs]
 
+        params = {'sample_rate': sorting.get_sampling_frequency()}
         if not save_path.is_dir():
-            save_path.mkdirs()
-        np.save(save_path /'spike_times.npy', spike_times[:, np.newaxis].astype(int))
+            save_path.mkdir(parents=True)
+        write_python(save_path / 'params.py', params)
+        np.save(save_path / 'spike_times.npy', spike_times[:, np.newaxis].astype(int))
```

Comment:
Personnaly I prefer to be totally explicit on dtpe and put 'int64'

Reply:
I can make it int64